spec/models/guide_tagger_job_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe GuideTaggerJob do
+  let(:api_double) { double(:publishing_api) }
+
+  before do
+    stub_const("PUBLISHING_API", api_double)
+  end
+
+  describe "#batch_perform_later" do
+    it "adds the topic to all guides" do
+      topic = create(:topic, :with_some_guides)
+      guide_ids = topic.tree.map {|t| t["guides"]}.flatten
+      guide_content_ids = Guide.where(id: guide_ids).pluck(:content_id)
+      guide_content_ids.each do |guide_content_id|
+        expect(api_double).to receive(:patch_links)
+          .with(
+            guide_content_id,
+            links: { topics: [topic.content_id] },
+          )
+      end
+
+      GuideTaggerJob.batch_perform_later(topic)
+    end
+  end
+end
```

Comment:
OOooh never mind I checked `git blame` and I'm gonna have to retract my burn. Nice work @tadast